dotnet-test-samples/sqs-lambda/src/template.yaml

```diff
@@ -0,0 +1,56 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: >
+  Sample SAM Template for sqs-lambda
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 10
+    MemorySize: 128
+
+Resources:
+  ProcessEmployeeFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: ./SqsEventHandler/
+      Handler: SqsEventHandler::SqsEventHandler.ProcessEmployeeFunction::Handler
+      Runtime: dotnet6
+      Architectures:
+        - x86_64
+      MemorySize: 256
```

Comment:
Setting memory twice (once in globals, once here)

Reply:
fixed